pubspec.yaml

```diff
@@ -25,17 +25,39 @@ dependencies:
     sdk: flutter
 
 
-  # The following adds the Cupertino Icons font to your application.
-  # Use with the CupertinoIcons class for iOS style icons.
+  # UI
   cupertino_icons: ^1.0.3
+  flutter_svg: ^0.22.0
+  nstack:
+    git:
+      url: git://github.com/nstack-io/flutter-sdk.git
+      #ref: v0.2.5
 
+  # Net
   dio: ^4.0.0
-  flutter_svg: ^0.22.0
+  jaguar_jwt: ^3.0.0
+
+  # Persistence
+  prefs: ^3.3.0
```

Comment:
anything this packages brings to the table to prefer it over Google's `shared_preferences`?